provisioning/service/tests/TestAssert.cs

```diff
@@ -104,16 +109,41 @@ public static void assertJObject(JObject expectedJObject, JObject actualJObject)
                 Assert.IsNotNull(acturalValue, FormatExpectedActual(expectedPair.Key + ":" + expectedPair.Value, "null"));
                 if (expectedPair.Value.Type == JTokenType.Object)
                 {
-                    Assert.IsTrue(acturalValue is JTokenType, 
-                        FormatExpectedActual(expectedPair.Value.ToString(), acturalValue.ToString()));
                     assertJObject((JObject)expectedPair.Value, (JObject)acturalValue);
                 }
+                else if (expectedPair.Value.Type == JTokenType.Array)
+                {
+                    assertJArray((JArray)expectedPair.Value, (JArray)acturalValue);
+                }
                 else
                 {
                     Assert.AreEqual(expectedPair.Value, acturalValue, 
                         FormatExpectedActual(expectedPair.Key + ":" + expectedPair.Value, expectedPair.Key + ":" + acturalValue));
                 }
             }
         }
+
+        public static void assertJArray(JArray expectedJObject, JArray actualJObject)
```

Comment:
Same as above: method should start with caps.

Both of these are incorrectly named. Instead they should follow `Assert.AreEqual` like `TestAssert.AreEqual(JArray, JArray)`.

Reply:
Done